lib/server/server.js

```diff
@@ -170,7 +170,16 @@ addDataSources({
   sourcePath: servicePath
 })
 
-const govukFrontendVersion = require(path.join(serviceNodeModules, 'govuk-frontend', 'package.json')).version
+const govukFrontentPath = path.join(serviceNodeModules, 'govuk-frontend')
+// Patch govukHint to allow block-level elements
+const hintTemplatePath = path.join(govukFrontentPath, 'components', 'hint', 'template.njk')
```

Comment:
Reminder for us to discuss how this change can work with my branch

Reply:
Typo: `frontent`

Reply:
Fixed